app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx

```diff
@@ -15,6 +17,13 @@ const labels = {
   [txTypes.REVOKED]: <T id="ticket.status.multiple.revoked" m="revoked" />
 };
 
+const qrHashes = [];
+const txPerQR = 15;
+// TODO: Use translated phrase or loading gif.
+const defaultLoading = "Loading...";
```

Comment:
this should be a boolean(true/false) - component should decide which message/content to show when `loading=true`.

Also in react you don't use global variables to store data.
Instead you should use mainly `useState` or other react hook.

Reply:
react hooks intro: https://reactjs.org/docs/hooks-intro.html
useState is in the first example.